www/src/Services/WebApi.js

```diff
@@ -639,7 +639,7 @@ async function getHeldPins(abortSignal) {
 		});
 		return response.data;
 	} catch (error) {
-		if (error?.code === 'ERR_CANCELED') return { canceled: true };
+		if (error.message === 'ERR_CANCELED') return { canceled: true };
```

Comment:
this constant is used only here so i assume it was coming from axios

Reply:
We should probably use fetch error 'AbortError' instead of creating our own/mimic axios